registration/scripts/directives.js

```diff
@@ -56,7 +63,13 @@
                             if (angular.isDefined(deferred)) {
                                 deferred.notify('reject', response);
                             }
+                            apiError = true;
                         });
+                    } else {
+                        //check there are no other error messages on the screen from the api
+                        if (!apiError) {
+                            scope.emptyEmail = true;
+                        }
```

Comment:
Yes it's a good question whether we should be doing any front-end validation at all, or leaving this all up to the API end? The problem this was trying to fix was that there was no error response from the API submitting a blank email address. Perhaps this should be fixed in the API?

I seem to recall having this conversation for another project. @maxpeterson Do you remember what we decided about this?